pandas/tests/arrays/test_integer.py

```diff
@@ -349,8 +349,8 @@ def test_error(self, data, all_arithmetic_operators):
             msg = (
                 r"(:?can only perform ops with numeric values)"
                 r"|(:?cannot perform .* with this index type: DatetimeArray)"
-                r"|(:?Addition/subtraction of integers and integer-arrays"
-                r" with DatetimeArray is no longer supported. *)"
+                r"|(:?Addition/subtraction of integers and integer-arrays "
+                r"with DatetimeArray is no longer supported. *)"
```

Comment:
this whitespace fix here is fine.  out of curiosity, is the regex group stuff really necessary?  i.e. the ":?"s

Reply:
It doesn't seems like it.